### PR TITLE
i18n(fr) Update `microsoft-azure.mdx`

### PR DESCRIPTION
**Pull request by McFlyPartages**
#### Description (required)

Update with #5057

### CODE REVIEW

File: src/content/docs/fr/guides/deploy/microsoft-azure.mdx [hunk 1]
```diff
@@ -21,6 +22,24 @@ Pour suivre ce guide, vous aurez besoin de :
...
+
+```
+  "engines": {
+    "node": ">=18.0.0"
+  },
+```
+
+## Ressources officielles
+
+- [Documentation de Microsoft Azure Static Web Apps](https://learn.microsoft.com/en-us/azure/static-web-apps/)
```
Comment (dreyfus92) [inline src/content/docs/fr/guides/deploy/microsoft-azure.mdx:41]:
```suggestion
- [Documentation de Microsoft Azure Static Web Apps](https://learn.microsoft.com/fr-fr/azure/static-web-apps/
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Bon boulot @McFlyPartages, voici un suggestion mineur.

**Review by thomasbnt — APPROVED** [review]
Lgtm!

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Amazing work 🚀

**Pull request merged**